aip/general/0203.md

```diff
@@ -220,6 +226,14 @@ There are some changes that *are* backwards compatible, which are as follows:
 
 ## Rationale
 
+## Identifier field behavior
+
+Resource names, the primary identifier for any compliant resource, are never
```

Comment:
nit: In my mind it would also help to rationalize this is by describing why we cannot use both "OUTPUT_ONLY" and "IMMUTABLE" instead to correctly describe this behavior, it's obvious to us but might help others. WDYT?

Reply:
Good idea. I've tried to do so. It ended up being a little bit more of a rewrite, ptal.